extensions/sql-migration/src/constants/strings.ts

```diff
@@ -610,6 +622,116 @@ export const RESOURCE_GROUP_DESCRIPTION = localize('sql.migration.resource.group
 export const DATA_UPLOADED_INFO = localize('sql.migration.data.uploaded.info', "Comparison of the actual amount of data read from the source and the actual amount of data uploaded to the target.");
 export const COPY_THROUGHPUT_INFO = localize('sql.migration.copy.throughput.info', "Data movement throughput achieved during the migration of your database backups to Azure. This is the rate of data transfer, calculated by data read divided by duration of backups migration to Azure.");
+export const SERVICE_SELECTION_LOCATION_MESSAGE = localize('sql.migration.service.selection.location.msg', "Please select the location of your database backup files before continuing.");
+
+// Validate IR dialog
+export const VALIDATION_STATE_CANCELED = localize('sql.migration.validation.state.canceled', "Canceled");
+export const VALIDATION_STATE_PENDING = localize('sql.migration.validation.state.pending', "Pending");
+export const VALIDATION_STATE_RUNNING = localize('sql.migration.validation.state.running', "Running");
+export const VALIDATION_STATE_SUCCEEDED = localize('sql.migration.validation.state.succeeded', "Succeeded");
+export const VALIDATION_STATE_FAILED = localize('sql.migration.validation.state.failed', "Failed");
+
+export const VALIDATE_IR_DONE_BUTTON = localize('sql.migration.validate.ir.done.button', "Done");
+export const VALIDATE_IR_HEADING = localize('sql.migration.validate.ir.heading', "We are validating the following:");
+export const VALIDATE_IR_START_VALIDATION = localize('sql.migration.validate.ir.start.validation', "Start validation");
+export const VALIDATE_IR_STOP_VALIDATION = localize('sql.migration.validate.ir.stop.validation', "Stop validation");
+export const VALIDATE_IR_COPY_RESULTS = localize('sql.migration.validate.ir.copy.results', "Copy validation results");
+export const VALIDATE_IR_RESULTS_HEADING = localize('sql.migration.validate.ir.results.heading', "Validation step details");
+export const VALIDATE_IR_VALIDATION_COMPLETED = localize('sql.migration.validate.ir.validation.completed', "Validation completed successfully.");
+export const VALIDATE_IR_VALIDATION_CANCELED = localize('sql.migration.validate.ir.validation.camceled', "Validation check canceled");
+
+export function VALIDATE_IR_VALIDATION_COMPLETED_ERRORS(msg: string): string {
+	return localize(
+		'sql.migration.validate.ir.completed.errors',
+		"Validation completed with the following error(s):{0}{1}", EOL, msg);
+}
+export function VALIDATE_IR_VALIDATION_STATUS(state: string | undefined, errors?: string[]): string {
+	const status = state ?? '';
+	if (errors && errors.length > 0) {
+		return localize(
+			'sql.migration.validate.ir.status.errors',
+			"Validation status: {0}{1}{2}", status, EOL, errors.join(EOL));
+	} else {
+		return localize(
+			'sql.migration.validate.ir.status',
+			"Validation status: {0}", status);
+	}
+}
+
+export function VALIDATE_IR_VALIDATION_STATUS_ERROR_COUNT(state: string | undefined, errorCount: number): string {
```

Comment:
@brian-harris thanks for updating this string, it does appear correctly now but I think it can still be improved a little bit. Currently it shows up as such: 
![image](https://user-images.githubusercontent.com/11844501/207948001-523cca8c-45cd-4af0-a75e-8963be337eea.png)

I think the count needs to go before the word "error(s)"